src/github/SyftGithubAction.ts

```diff
@@ -69,6 +69,34 @@ export function getArtifactName(): string {
   return `${repo}-${job}${stepName}.${extension}`;
 }
 
+/**
+ * Execute directly for linux & macOS and use WSL for Windows
+ * @param cmd command to execute
+ * @param args command args
+ * @param options command options
+ */
+async function execute(
+  cmd: string,
+  args: string[],
+  options?: exec.ExecOptions
+) {
+  if (process.platform === "win32") {
```

Comment:
Should we add a test to `tests/SyftGithubAction.test.ts` to just make it plain the transformation expected by the replacement here?

It would also be a good spot where we can tack on any extra regression cases if we find weird edge cases in the future or if we need to change up the replace logic